docs/wiki/framework/tagging-framework.md

```diff
@@ -59,3 +63,24 @@ class ACE_Tags {
 2  | `"ACE_SpraypaintRed"` | Required item to have in the inventory to be able to spray your tag
 3  | `["tagTexture1.paa", "tagTexture2.paa"]` | List of texture variants (one is randomly selected when tagging)
 4  | `"icon.paa"` | Icon being displayed in the interaction menu
+
+
+### 2.2 Tags in description.ext
+
+Tags can also be configured in description.ext like shown above.
+File Paths will be relative to your mission, if you want to define Tags inside description.ext but use Addon paths for `Texture`/`Material`/`TagModel` you need to prefix the path with `@`.
```

Comment:
ref, this is a semi-standard used in vanilla
>Since Arma 3 v1.49.131710 it is possible to define AddOn sounds in mission config. In order to make engine look for the sound in AddOn, the sound path must start with @ (instead of \) for example:

Reply:
Should that be written down in docs somehow? Don't know how